### ISSUE TITLE
OpenGL support? [qemu]

### ISSUE THREAD

**Issue by demberto** [msys2/MINGW-packages#8506]
Why is OpenGL support disabled? There do exist Mesa drivers for Windows (I don't know if this is related or not)

**Comment by ZachBacon** [comment]
> Why is OpenGL support disabled? There do exist Mesa drivers for Windows (I don't know if this is related or not)

Because we have sdl2? Iirc by default sdl2 with qemu defaults to its OpenGL renderer. However as for why it's disabled, usually if OpenGL headers are found, then it most likely would of enabled automatically. But if not then I think it could be enabled.

**Comment by demberto** [comment]
OpenGL supports needs to be added while compiling and no, `-display sdl` and `-display sdl,gl=on` makes a huge difference.

**Comment by ZachBacon** [comment]
> OpenGL supports needs to be added while compiling and no, `-display sdl` and `-display sdl,gl=on` makes a huge difference.

Yes I'm aware of that, but I'm saying by default sdl2 does use OpenGL. I'm not saying there's a switch for qemu and SDL to use it.

Either way if you want to go ahead and enable it in the pkgbuild, please feel free to submit a pull request

**Comment by ZachBacon** [comment]
so I tested opengl support in qemu, it's trying to make use of EGL which at the moment can only be achieved via angle, which has issues compiling on mingw-w64, so I'm assuming that's why it's disabled

**Comment by demberto** [comment]
So there's no other way? I mean there are [pre-built mesa drivers for windows](https://github.com/pal1000/mesa-dist-win) which do have EGL libraries (`libGLESv1_CM.dll` and `libGLESv2.dll`) and they do have mingw builds. I'm sorry if this isn't related

**Comment by ZachBacon** [comment]
I don't know about the others, but I don't want something as large as Mesa as a dependency for qemu that doesn't play nice sometimes. I've had its OpenGL module not play well with nvidia GPUs on windows.

**Comment by lazka** [comment]
We have angle now and qemu depends on it. Does this mean this issue is fixed?
